internal/store/utils.go

```diff
@@ -215,3 +216,11 @@ func mergeKeyValues(keyValues ...[]string) (keys, values []string) {
 
 	return keys, values
 }
+
+// convertValueToFloat64 converts a resource.Quantity to a float64.
+func convertValueToFloat64(q *resource.Quantity) float64 {
+	if q.Value() > resource.MaxMilliValue {
```

Comment:
Could we remove if-else and use `float64(q.MilliValue() / 1000)`?

Reply:
may be not. q.MilliValue() will cause overflow when q.Value() is larger than maxInt64/1000. for example, 1000TB will return a negative number.

Reply:
Ack.